src/index.js

```diff
@@ -0,0 +1,93 @@
+import './style.css';
+import Icon from './menu.png';
+import Reload from './reload.png';
+import Enter from './to-left.png';
+
+function component() {
+  const tasks = [
+    {
+      description: 'Clean room',
+      completed: false,
+      index: 1,
+    },
+    {
+      description: 'Cook lunch',
+      completed: false,
+      index: 2,
+    },
+  ];
```

Comment:
- [ ] **Required:** At the moment your to do list do not display according to the index, kindly fix by sorting them by their indexes. Eg. `Array.sort((a, b) => a.index - b.index);`.